### DISCUSSION TITLE
I keep getting this crrash log with some mods and it seems to be something wrong with Oh the biomes you'll go.but when i remove it it just crashes without any log or anything?

### DISCUSSION THREAD

**Discussion by BB4BB** [FabricMC/fabric-api#2214] [Player support] [answered]
---- Minecraft Crash Report ----
// I bet Cylons wouldn't have this problem.

Time: 2022-05-12 17:43
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'byg'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:452)
	at net.minecraft.client.main.Main.main(Main.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	Suppressed: java.lang.IllegalStateException: java.io.IOException: BYG config found at: "C:\Users\anton\AppData\Roaming\.minecraft\config\byg\overworld-biomes.json" could not be read. The fastest solution is to rename this failed file and let a new file generate from BYG and replace the fields in the new file with the broken file's fields.
		at potionstudios.byg.config.json.OverworldBiomeConfig.readConfig(OverworldBiomeConfig.java:62)
		at potionstudios.byg.config.json.OverworldBiomeConfig.getConfig(OverworldBiomeConfig.java:34)
		at potionstudios.byg.entry.BYGTerraBlenderEntry.onTerraBlenderInitialized(BYGTerraBlenderEntry.java:35)
		at terrablender.core.TerraBlenderFabric.lambda$onInitialize$1(TerraBlenderFabric.java:46)
		at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
		at terrablender.core.TerraBlenderFabric.onInitialize(TerraBlenderFabric.java:41)
		at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
		... 11 more
	Caused by: java.io.IOException: BYG config found at: "C:\Users\anton\AppData\Roaming\.minecraft\config\byg\overworld-biomes.json" could not be read. The fastest solution is to rename this failed file and let a new file generate from BYG and replace the fields in the new file with the broken file's fields.
		at potionstudios.byg.util.BYGUtil.configFileFailureException(BYGUtil.java:97)
		at potionstudios.byg.config.json.OverworldBiomeConfig.lambda$readConfig$3(OverworldBiomeConfig.java:60)
		at java.base/java.util.Optional.orElseThrow(Optional.java:403)
		at potionstudios.byg.config.json.OverworldBiomeConfig.readConfig(OverworldBiomeConfig.java:60)
		... 17 more
Caused by: java.lang.IllegalStateException: java.io.IOException: BYG config found at: "C:\Users\anton\AppData\Roaming\.minecraft\config\byg\byg-end-biomes.json" could not be read. The fastest solution is to rename this failed file and let a new file generate from BYG and replace the fields in the new file with the broken file's fields.
	at potionstudios.byg.common.world.biome.end.EndBiomesConfig.readConfig(EndBiomesConfig.java:97)
	at potionstudios.byg.common.world.biome.end.EndBiomesConfig.getConfig(EndBiomesConfig.java:59)
	at potionstudios.byg.common.world.biome.end.EndBiomesConfig.getConfig(EndBiomesConfig.java:54)
	at potionstudios.byg.BYG.commonLoad(BYG.java:58)
	at potionstudios.byg.BYGFabric.onInitialize(BYGFabric.java:61)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 11 more
Caused by: java.io.IOException: BYG config found at: "C:\Users\anton\AppData\Roaming\.minecraft\config\byg\byg-end-biomes.json" could not be read. The fastest solution is to rename this failed file and let a new file generate from BYG and replace the fields in the new file with the broken file's fields.
	at potionstudios.byg.util.BYGUtil.configFileFailureException(BYGUtil.java:97)
	at potionstudios.byg.common.world.biome.end.EndBiomesConfig.lambda$readConfig$11(EndBiomesConfig.java:95)
	at java.base/java.util.Optional.orElseThrow(Optional.java:403)
	at potionstudios.byg.common.world.biome.end.EndBiomesConfig.readConfig(EndBiomesConfig.java:95)
	... 16 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:452)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		AMSIExt.dll:McAfee Cloud AV ThirdParty Ext:26.0.144 9c8b9018 smoldering cucumber:McAfee, LLC
		COMCTL32.dll:Bibliotek för användargränssnittskontroller:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:32-bitars kryptografi-API:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DLL-fil med API för DNS-klient:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Klient-DLL för Windows NT BASE API:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Klient-DLL för Windows NT BASE API:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		MSASN1.dll:ASN.1 Runtime APIs:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.2204.4 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE för Windows:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		OleAut32.dll:OLEAUT32.DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Körning av RPC (Remote Procedure Call):10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:DLL-fil för Windows-gränssnittet:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Fleranvändarklient-DLL-fil för Windows:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		WINTRUST.dll:Microsoft Trust Verification APIs:10.0.22000.527 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:DLL-filen för Windows Socket 2.0 32-Bit:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.22000.593 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Bibliotek för kryptografiska primitiver i Windows:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.22000.376 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM för Windows:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		cryptnet.dll:Crypto Network Related API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:Tjänsten DHCP Client:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:DHCPv6-klient:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:Användarläges-API för FWP/IPsec:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.22000.527 (WinBuild.160101.0800):Microsoft Corporation
		gpapi.dll:API för grupprincipklient:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		imagehlp.dll:Windows NT Image Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.1.0:Microsoft
		jimage.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		jna7647442194597337062.dll:JNA native library:6.1.1:Java(TM) Native Access (JNA)
		jvm.dll:OpenJDK 64-Bit server VM:17.0.1.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.22000.71 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		mfehida.dll:McAfee Driver Communication:SYSCORE.22.2.0.148:McAfee, LLC
		mfemmsa.dll:McAfee Management Service API:SYSCORE.22.2.0.148:McAfee, LLC
		msvcp140.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Tjänstprovider för Microsoft Windows Sockets 2.0:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:Provider för e-postnamngivnings-shim:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt-router:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:DLL-fil för NT Layer:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:DLL-fil med systemprestandaobjekt:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP-namnområdesprovider:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		powrprof.dll:Hjälp-DLL för energiprofil:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		svml.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		win32u.dll:Win32u:10.0.22000.593 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage-API:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows bastyps-DLL:10.0.22000.613 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.22000.41 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.18.1
	Minecraft Version ID: 1.18.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1162878400 bytes (1109 MiB) / 2013265920 bytes (1920 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 6
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-9400F CPU @ 2.90GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 10
	Microarchitecture: Coffee Lake
	Frequency (GHz): 2,90
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 6
	Graphics card #0 name: NVIDIA GeForce GTX 1650
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095,00
	Graphics card #0 deviceId: 0x1f82
	Graphics card #0 versionInfo: DriverVersion=30.0.15.1215
	Memory slot #0 capacity (MB): 8192,00
	Memory slot #0 clockSpeed (GHz): 2,67
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 16226,06
	Virtual memory used (MB): 10523,07
	Swap memory total (MB): 8113,03
	Swap memory used (MB): 530,64
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		additionalstructures: Additional Structures 3.0.3
		advancementplaques: Advancement Plaques 1.4.3
		alloy_forgery: Alloy Forgery 2.0.9+1.18
		amecsapi: Amecs API 1.3.6+mc1.18-pre1
		awesomedungeon: Awesome Dungeon 2.0.8
		balm-fabric: Balm 2.4.3+0
		bclib: BCLib 1.3.5
		betterdungeons: YUNG's Better Dungeons 1.18-1.0
		betterend: Better End 1.0.3
		bettermineshafts: YUNG's Better Mineshafts 1.18.1-1.0
		betternether: Better Nether 6.0.12
		betterstrongholds: YUNG's Better Strongholds 1.18.1-1.0.1
		blur: Blur (Fabric) 2.4.1
		byg: BYG 1.3.5.13
		capybara: Capybara 1.0.1
		cloth-basic-math: cloth-basic-math 0.6.1
		cloth-config: Cloth Config v6 6.2.62
		clumps: Clumps 8.0.0+5
		coat: Coat 1.0.0-beta.17+mc1.18-pre1
		com_electronwill_night-config_core: core 3.6.4
		com_electronwill_night-config_toml: toml 3.6.4
		com_eliotlash_mclib_mclib: mclib 18
		com_eliotlash_molang_molang: molang 18
		com_typesafe_config: config 1.4.1
		confabricate: confabricate 2.2.0-SNAPSHOT+4.1.1
		croptopia: Croptopia 1.8.3
		dummmmmmy: MmmMmmMmmMmm 1.18-1.5.0
		dungeons_arise: When Dungeons Arise 2.1.49d-fabric
		dynamicsoundfilters: Dynamic Sound Filters 1.3.0+1.18
		enchant_giver: Enchant Giver 1.1.0
		endgoblintraders: End Goblin Traders 1.4.4
		fabric: Fabric API 0.46.6+1.18
		fabric-api-base: Fabric API Base 0.4.2+d7c144a865
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.5.3+d7c144a865
		fabric-biome-api-v1: Fabric Biome API (v1) 6.0.2+d7c144a865
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.10+3ac43d9565
		fabric-command-api-v1: Fabric Command API (v1) 1.1.7+d7c144a865
		fabric-commands-v0: Fabric Commands (v0) 0.2.6+b4f4f6cd65
		fabric-containers-v0: Fabric Containers (v0) 0.1.19+d7c144a865
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.9+d7c144a865
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.9+3ac43d9565
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.10+a1d9bbf565
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.6+d7c144a865
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.17+d7c144a865
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.9+d7c144a865
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.11+d7c144a865
		fabric-item-api-v1: Fabric Item API (v1) 1.3.1+691a79b565
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.7+3ac43d9565
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.9+d7c144a865
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.7+b4f4f6cd65
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.13+713c266865
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.9+d7c144a865
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.7+d7c144a865
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.12+b4f4f6cd65
		fabric-models-v0: Fabric Models (v0) 0.3.4+d7c144a865
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.19+d7c144a865
		fabric-networking-v0: Fabric Networking (v0) 0.3.6+b4f4f6cd65
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.5+737332ce65
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.13+d7c144a865
		fabric-particles-v1: Fabric Particles (v1) 0.2.10+526dc1ac65
		fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.2+ad01bfbd65
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.11+b0b66fc365
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.15+6825030165
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.10+b4f4f6cd65
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.5+d7c144a865
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.19+3ac43d9565
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.12+b4f4f6cd65
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.6+713c266865
		fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 1.0.2+d7c144a865
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.15+8906aafd65
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.8+d7c144a865
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.12+d7c144a865
		fabric-structure-api-v1: Fabric Structure API (v1) 2.1.3+d7c144a865
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.9+d7c144a865
		fabric-textures-v0: Fabric Textures (v0) 1.0.10+3ac43d9565
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.9+fb3b57b465
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.10+c329913d65
		fabricloader: Fabric Loader 0.13.3
		geckolib3: Geckolib 3.0.33
		goblintraders: Goblin Traders 1.5
		graveyard: The Graveyard 1.6
		io_leangen_geantyref_geantyref: geantyref 1.3.11
		iris: Iris 1.2.0-pre
		java: OpenJDK 64-Bit Server VM 17
		kirin: Kirin UI 1.10.0-beta.2
		libraryferret: Library Ferret 1.0.3
		magna: Magna 1.7.0-1.18-pre1
		mcda: MC Dungeons Armors 2.0.2
		mcdw: MC Dungeons Weapons 4.0.4
		midnightlib: MidnightLib 0.3.1
		minecraft: Minecraft 1.18.1
		modmenu: Mod Menu 3.0.1
		mostructures: Mo' Structures 1.3.1+1.18.1
		mousewheelie: Mouse Wheelie 1.8.8+mc1.18-pre5
		mythicmetals: Mythic Metals 0.12.2
		omega-config: OmegaConfig 1.2.3-1.18.1
		org_anarres_jcpp: jcpp 1.4.14
		org_joml_joml: joml 1.10.2
		org_slf4j_slf4j-api: slf4j-api 1.7.12
		org_spongepowered_configurate-core: configurate-core 4.1.1
		org_spongepowered_configurate-extra-dfu4: configurate-extra-dfu4 4.1.1
		org_spongepowered_configurate-gson: configurate-gson 4.1.1
		org_spongepowered_configurate-hocon: configurate-hocon 4.1.1
		owo: oωo 0.5.2+1.18
		presencefootsteps: Presence Footsteps 1.4.0
		reach-entity-attributes: Reach Entity Attributes 2.1.1
		roughlyenoughitems: Roughly Enough Items 7.4.456
		satin: Satin 1.7.0
		seedfix: Seed Fix 1.0.0
		sodium: Sodium 0.4.0-alpha6+build.14
		spark: spark 1.8.6
		terrablender: TerraBlender 1.1.0.77
		terralith: Terralith for Fabric 2.0.12
		thonkutil: ThonkUtil 2.7.1+1.18
		thonkutil-base: ThonkUtil Base 1.5.1+31ad4eb1c8
		thonkutil-capes-v1: ThonkUtil Capes (v1) 1.2.2+befce073c8
		thonkutil-coords-v1: ThonkUtil Coords (v1) 1.0.8+526fd6e5c8
		thonkutil-customization-v1: ThonkUtil Customization (v1) 1.0.3+526fd6e5c8
		thonkutil-legacy: ThonkUtil Legacy 1.0.9+6fa117f8c8
		thonkutil-modchecker-v1: ThonkUtil ModChecker (v1) 1.0.0+aebf7ba0c8
		thonkutil-potions-v0: ThonkUtil Potions (v0) 1.3.1+befce073c8
		thonkutil-trades-v1: ThonkUtil Trades (v1) 1.1.8+d77526a2c8
		toms_storage: Tom's Simple Storage Mod 1.1.19
		tweed4_annotated: tweed4_annotated 1.3.0+mc1.18-pre2
		tweed4_base: tweed4_base 1.5.0+mc1.18-pre2
		tweed4_data: tweed4_data 1.2.1+mc1.18-pre2
		tweed4_data_hjson: tweed4_data_hjson 1.1.0+mc1.18-pre2
		tweed4_tailor_coat: tweed4_tailor_coat 1.1.2+mc1.18-pre2
		tweed4_tailor_screen: tweed4_tailor_screen 1.1.1+mc1.18-pre2
		waystones: Waystones 9.0.4
		yungsapi: YUNG's API 1.18-Fabric-36
		yungsbridges: YUNG's Bridges 1.18.1-1.0
		yungsextras: YUNG's Extras 1.18.1-1.0
	Launched Version: fabric-loader-0.13.3-1.18.1
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Unknown
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>

**Comment by modmuss50** [comment, marked as answer]
Seems to be an issue with your BYG config file, the error explains how to fix it.

`"C:\Users\anton\AppData\Roaming.minecraft\config\byg\overworld-biomes.json" could not be read. The fastest solution is to rename this failed file and let a new file generate from BYG and replace the fields in the new file with the broken file's fields.`

**Reply by BB4BB** [reply]
Thanks a lot, it helped so much!
